### PR TITLE
Snapshot on multi_disks under transaction mode

### PR DESCRIPTION
**Pull request by aliang123**
blockdev_snapshot_multi_disks:snapshot on multi_disks under transaction mode

Signed-off-by: Aihua Liang <aliang@redhat.com>

id: 1802028

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_snapshot_multi_disks.q35: PASS (107.96 s)

**Comment by aliang123** [comment]
@meet-lucky, @ngu-niny, @xutian  please help to review, thanks

File: qemu/tests/blockdev_snapshot_multi_disks.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import logging
+
+from virttest import utils_disk
+from virttest import data_dir
+from virttest import error_context
+
+from provider.blockdev_snapshot_base import BlockDevSnapshotTest
+from provider import backup_utils
```
Comment (xutian) [inline qemu/tests/blockdev_snapshot_multi_disks.py]: change import order

from provider import backup_utils
from provider.blockdev_snapshot_base import BlockDevSnapshotTest

File: qemu/tests/blockdev_snapshot_multi_disks.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            logging.error(str(error))
+
+
+def run(test, params, env):
+    """
+    Backup VM disk test when VM reboot
+
+    1) start VM with system disk
+    2) create target disk with qmp command
+    3) load stress in guest
```
Comment (xutian) [inline qemu/tests/blockdev_snapshot_multi_disks.py]: where step to load stress?

Comment (aliang123) [reply qemu/tests/blockdev_snapshot_multi_disks.py]: Hi, have updated, please help to review again.

File: qemu/tests/blockdev_snapshot_multi_disks.py (hunk 1 above)
Comment (aliang123) [reply qemu/tests/blockdev_snapshot_multi_disks.py]: Hi, have updated, please help to review again.

**Comment by meet-lucky** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_snapshot_multi_disks.q35: PASS (77.66 s)

Acked-by:  Qian Dong  qdong@redhat.com

**Comment by xutian** [comment]
LGTM

**Comment by ngu-niny** [comment]
Test Pass:

'1-Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.blockdev_snapshot_multi_disks' Finshed 2020-02-13 21:37:12 2020-02-13 21:41:12 PASS 240.10385

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**